resources/views/dispatch/show.blade.php

```diff
@@ -97,14 +97,18 @@
                         <div class="form-group card-text row mb-2">
                             <label class="col-5 col-form-label">{{ $fpl['origin']['iata_code'] }}</label>
                             <div class="col-7">
-                                <input type="text" class="form-control form-control-sm" id="depstand">
+                                <form onsubmit="addStand(event)">
```

Comment:
I'm gonna add a save button to this because otherwise the only way to submit the form is the return key

Reply:
Resolved by https://github.com/MultiCrew/copilot/pull/125/commits/d2abe6b36b160ecfdfb238c4dff15b520b4ae068